### ISSUE TITLE
Notification when production calendar changes

### ISSUE THREAD

**Issue by svevang** [PRX/cms.prx.org#517]
A feature request that came out of discussion re: production calendar and ad-ops.  Create a notification that occurs when the production calendar changes.

**Comment by cavis** [comment]
Does that mean:

- Notify when _any_ podcast story has a `published_at` or `released_at` changed?
- Notify only when existing stories are updated?
- Notify only when released_at is within X days of now?
- Some combination of those?

**Comment by svevang** [comment]
Good questions, and worth discussing in detail. This idea came up in the context of adjusting download inventory numbers (rerunning forecasts) in the case that there was a change in the production calendar.

**Comment by brandonhundt** [comment]
@svevang I've discussed this with Ad/Ops, the ticket is just against publish.  

Do you see any differences here?  https://github.com/PRX/publish.prx.org/issues/581

**Comment by svevang** [comment]
@brandonhundt Thanks for pointing that out.  https://github.com/PRX/publish.prx.org/issues/581 . feels closely related but a little different.  This is less about alarms and more about having general insight into the dynamics of the production calendar. 

The problem was related to the workflow around picking and running forecasts on CMS series.  Specifically, which series are ready to have forecasts run?  And, how many series have the production calendar completed?

I'm not sure what the solution looks like here, but @cavis per your earlier questions, I was imagining an event stream of all the calendar CRUD operations. If we integrated over that stream (by e.g. day) than perhaps we'd have a way to characterize 'movement' on the calendar ('series 123 create 44 calendar stories today').

That's the scenario, and a (probably bad) solution. The workflows are new here, so I'm using github to keep track of things as they happen..

**Comment by cavis** [comment]
Yeah, does feel like we should keep this in mind if we're going to update our AnnounceActions in CMS to include the previous activerecord data.
